spec/lib/spree/adyen/api_response_spec.rb

```diff
@@ -7,6 +7,12 @@
       body: "resultCode=Authorised&pspReference=1234567890",
     )
   end
+  let(:http_redirect) do
+    instance_double(
+      "Net::HTTPSuccess",
+      body: "resultCode=RedirectShopper&md=encryptedmd&paRequest=encryptedpaReq&issuerUrl=https%3A%2F%2Ftest.adyen.com%2Fhpp%2F3d%2Fvalidate.shtml&pspReference=1234567890"
```

Comment:
Don't really care but it would probably be more clear if this was something along the lines of
```ruby
require 'uri'
URI.encode_www_form({"resultCode"=>["RedirectShopper"],
                     "md"=>["encryptedmd"],
                     "paRequest"=>["encryptedpaReq"],
                     "issuerUrl"=>
                     ["https://test.adyen.com/hpp/3d/validate.shtml"],
                     "pspReference"=>["1234567890"]})
# [2] pry(main)* => "resultCode=RedirectShopper&md=encryptedmd&paRequest=encryptedpaReq&issuerUrl=https%3A%2F%2Ftest.adyen.com%2Fhpp%2F3d%2Fvalidate.shtml&pspReference=1234567890"

```